test/blockstore-test.js

```diff
@@ -281,7 +282,7 @@ describe('BlockStore', function() {
   });
 
   describe('FileBlockStore (Unit)', function() {
-    const location = '/tmp/.bcoin/blocks';
+    const location = testdir('bcoin');
```

Comment:
Was this necessary? `testdir()` is for integration/e2e tests that actually touch the disk, these are all unit tests.

Reply:
Could something like this work?

https://github.com/bcoin-org/bcoin/commit/3745f6539a14cbcdf193551100ab3d023a22507f

This is how I had it earlier but wasn't sure if we want to use `testdir` here. I will drop the global though, it should really be inside the unit tests.

Reply:
Will look into it a bit more.

Reply:
OK still not an issue to go back to test local imo.